packages/styles/src/mixins/list/_list-search-results.scss

```diff
@@ -0,0 +1,50 @@
+@use "sass:map";
+
+@import "../mixins";
+@import "./list-definitions";
+
+.#{$block} {
```

Comment:
this can be directly `.#{$block}--search-results` or if you decode to use the variable you declared in `packages/styles/src/mixins/list/_list-definitions.scss`, you can use `.#{$searchResultsList}`